backend/handlers/game.js

```diff
@@ -2,22 +2,94 @@
 
 // Imports
 const GameAPI = require("../services/GameAPI");
+const moment = require("moment-timezone");
 const responseUtil = require("../utilities/response");
 
 module.exports = {
-
-    // CREATE A NEW GAME //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-    Create: async function () {
         
+    // MAKE A NEW GAME  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    CreateGame: async function (event) {
+        try {
+            // if nothing was provided in the request, return a 204 HTTPS code (No content)
+            if (!event)
+                return responseUtil.Build(204, { Message: "No information was provided with the request!" });
+
+            // next parse the request
+            let request = JSON.parse(event.body);
+
+            // they need to have entered an email, username, and a password
+            if (!request.GameName)
+                throw new Error("Game Name Required!");
+
+            if (!request.Genre)
+                throw new Error("Game Genre Required!");
```

Comment:
If we want to worry about long term maintainability, it may be simpler to make an array of strings of the required attributes, and look through to see if any is missing. Not a required change, but something to think about in case we want to continue on this project beyond the SE class